src/store/utils.test.tsx

```diff
@@ -22,6 +22,14 @@ import { fetchAuthURLs } from 'terraso-client-shared/account/accountSlice';
 import { useFetchData } from 'terraso-client-shared/store/utils';
 import { render } from 'terraso-client-shared/tests/utils';
 
+type NotificationMessage = {
+  severity: string;
+  content: string;
+  params: {
+    error: string;
+  };
+};
+
```

Comment:
question: is there a reason we can't use the `Message` type from `notificationsSlice.ts`? and do you know why this test used to type correctly and no longer does? we shouldn't need to cast here because the values coming out of the store's state should be typed properly, and the cast makes the codebase more brittle. so i worry whether this update will also create type errors downstream. did you check whether `mobile-client` built against this version of `client-shared` compiles correctly?